src/assets/config_files/usecases.json

```diff
@@ -1553,9 +1565,12 @@
         },
         "next": "ta_form",
         "dataprotected": true,
+	"contributors": [
```

Comment:
Could you please indent this so it is in the same level as the others?

Reply:
Done now.